### PR TITLE
Cart 146 markup for unique users counter

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

File: dev/publicDev/index.jade [hunk 1]
```diff
@@ -12,6 +12,9 @@ html(lang='en-US', ng-app='app')
       footer
         p @ 2016 All rights reserved
         p Design by Juless Design
-    p.visitors Total {{vm.visitorsCount.value}} visitors
+    .visitors
+      p Hit
```
Comment (dmitrij-borchuk) [inline dev/publicDev/index.jade]: Remove "hit" and "page view" because we don't have this information

File: dev/publicDev/index.jade [hunk 2]
```diff
@@ -12,6 +12,9 @@ html(lang='en-US', ng-app='app')
       footer
         p @ 2016 All rights reserved
         p Design by Juless Design
-    p.visitors Total {{vm.visitorsCount.value}} visitors
+    .visitors
```
Comment (dmitrij-borchuk) [inline dev/publicDev/index.jade]: Remove this counter from all page except "home"

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/publicDev/index.jade (hunk 2 above)
Comment (RomanFedytskyi) [reply dev/publicDev/index.jade]: fixed

File: dev/publicDev/index.jade (hunk 1 above)
Comment (RomanFedytskyi) [reply dev/publicDev/index.jade]: fixed

**Review by dmitrij-borchuk — APPROVED** [review]

**Pull request merged**